Tests/TuistGeneratorTests/Utils/AbsolutePath+String.swift

```diff
@@ -0,0 +1,7 @@
+import Basic
+
+extension AbsolutePath: ExpressibleByStringLiteral {
```

Comment:
This was copied - as a few tests translate Strings > AbsolutePath for convenience